images/hubble-proto/Dockerfile

```diff
@@ -13,25 +13,24 @@ RUN /tmp/install-protoplugins.sh
 
 FROM docker.io/library/alpine:3.12
 
+WORKDIR /proto
+
 RUN apk add --no-cache \
-    curl \
-    bash \
-    make \
-    && true
+  curl \
+  bash \
+  make \
+  && true
 
 COPY install-glibc.sh /tmp/install-glibc.sh
 RUN /tmp/install-glibc.sh
 
 COPY install-protoc.sh /tmp/install-protoc.sh
 RUN /tmp/install-protoc.sh
 
-RUN mkdir /proto
-COPY --from=builder /proto/protoc-gen-go /proto
-COPY --from=builder /proto/protoc-gen-go-grpc /proto
-COPY --from=builder /proto/protoc-gen-go-json /proto
-
-COPY Makefile.docker /proto/Makefile
+COPY --from=builder /proto/protoc-gen-go .
+COPY --from=builder /proto/protoc-gen-go-grpc .
+COPY --from=builder /proto/protoc-gen-go-json .
 
-WORKDIR /source
```

Comment:
Will removing this final WORKDIR make a difference in any subsequent images using this Dockerfile @errordeveloper? It appeared non-essential but probably good to double-check.

Reply:
The `hubble-proto` container is invoked by `api/v1/Makefile` this way:
```
docker container run --rm \
        --volume $(CURDIR):/source \
        cilium/hubble-proto:${version}
```
Therefore, removing `WORKDIR /source` will break API proto generation.

Reply:
Thanks, @Rolinh for confirming, I'll re-add this as it seemed suspicious that it was just arbitrarily added but then nothing in this Dockerfile suggested it was being used.

Reply:
reintroduced as mentioned above by @Rolinh it's required.